db/blip_handler.go

```diff
@@ -747,10 +756,11 @@ func (bh *blipHandler) handleRev(rq *blip.Message) (err error) {
 
 	// Finally, save the revision (with the new attachments inline)
 	// If a conflict resolver is defined for the handler, write with conflict resolution.
+	forceAllowConflicts := newDoc.Deleted && (bh.conflictResolver != nil || bh.clientType == BLIPClientTypeSGR2)
```

Comment:
Can you just stick a comment above stating that if the doc is a tombstone, we want to allow conflicts when running sgr2.  'bh.conflictResolver != nil' represents an active sgr2, BLIPClientTypeSGR2 represents passive sgr2